strawberryfields/backends/tfbackend/circuit.py

```diff
@@ -249,6 +253,13 @@ def reset(self, **kwargs):
             self._batch_size = batch_size
             self._batched = batch_size is not None
 
+        if "dtype" in kwargs:
+            dtype = kwargs["dtype"]
+            if dtype is not None:
+                if not dtype in (tf.complex64, tf.complex128):
+                    raise ValueError("Argument 'dtype' must be a complex Tensorflow DType")
```

Comment:
Optional suggestion: you could make this code block slightly more concise by doing something like:

```python
self._dtype = kwargs.get("dtype", tf.complex64)

if self._dtype is not in (tf.complex64, tf.complex128):
    raise ValueError("Argument 'dtype' must be a complex Tensorflow DType")
```

Reply:
I like it, good to do similar for the other circuit args?